web/src/domain/game.ts

```diff
@@ -0,0 +1,52 @@
+import { Multimedia } from "./multimedia";
+import { Tag } from "./tag";
+
+/**
+ * Represents a game.
+ */
+export interface Game {
+  id: string;
+  publisherId: string;
```

Comment:
The `Game` schema does not contain the `publisherId`, but the actual `Publisher` schema
```suggestion
  publisher: Publisher;
```